messages/action-summary-viewer.md

```diff
@@ -1,3 +1,7 @@
+# common.streaming-logs-to
+
+**Streaming logs in real time to:**
```

Comment:
@jshackell-sfdc , Net-new message. Example of its use can be seen in the PR's description section.

Reply:
LGTM (although I assume you'll remove the ** around the message?  Or do you intend to leave that?  It looks odd to me...)